src/core/feed-core.js

```diff
@@ -278,6 +280,15 @@ function _getSortingSql(sort) {
   }
 }
 
+function _getTeamNameSql(cityId) {
+  return !cityId
+    ? `teams.name`
+    : `CASE
+        WHEN teams.city_id=${cityId} THEN teams.name
```

Comment:
Should never interpolate parameter directly into SQL string. This is prone to SQL injection attack